Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM ubuntu:latest
 LABEL maintainer="akshay.kumar758@webkul.com"
 
-ENV GOSU_VERSION 1.11
+ENV GOSU_VERSION=1.11
 
+RUN apt-get update && apt-get install -y adduser
 RUN adduser uvdesk -q --disabled-password --gecos ""
```

Comment:
This could also just be moved after `# Install base supplementary packages`  line 31 and line 6 is not needed